TheMET/TheMET/ViewControllers/CategoryViewController.swift

```diff
@@ -134,13 +134,18 @@ class CategoryViewController: UIViewController, UITableViewDelegate, UITableView
     }
     
     private func loadArtCellData(artID: ArtID, completion: @escaping (ArtCellData?) -> Void) {
-        self.metAPI.object(id: artID) {object in
-            guard let object = object else {
-                completion(nil)
-                return
+        self.metAPI.object(id: artID) {objectResult in
+//            guard let objectResult = objectResult else {
+//                completion(nil)
+//                return
+//            }
+            switch objectResult {
+            case .failure:
+                self.contentStatus = .failed
```

Comment:
as I se the code worked differently in this case before. It was executing `completion`